blockchain-albatross/src/blockchain.rs

```diff
@@ -1074,7 +1074,15 @@ impl Blockchain {
 
     pub fn get_epoch_transactions(&self, epoch: u32, txn_option: Option<&Transaction>) -> Option<TransactionsIterator> {
         let first_block = policy::first_block_of(epoch);
-        let first_block = self.chain_store.get_block_at(first_block, true, txn_option)?;
+        let first_block_opt = self.chain_store.get_block_at(first_block, true, txn_option);
```

Comment:
I would have preferred to use `.or_else(|| debug!(...); None)?;`, but this is fine, too.